tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -22,20 +23,358 @@
  *
  * @internal
  *
- * @covers \PhpCsFixer\AbstractAlignFixerHelper
```

Comment:
Are those classes still covered by other tests? Even if deprecated, they are supposed to work until v3.0... but is it worth it?

Reply:
For that you would need the current Bin.Op.Fixer and its test back, you want to maintain those as well as part of the test suit?

Reply:
Not really :p